src/v/raft/group_configuration.cc

```diff
@@ -142,7 +142,7 @@ group_configuration::group_configuration(
   std::vector<model::broker> brokers, model::revision_id revision)
   : _brokers(std::move(brokers))
   , _revision(revision) {
-    _current.voters.resize(brokers.size());
+    _current.voters.resize(_brokers.size());
```

Comment:
@mmaslankaprv is the intention here to `resize` (in which case current voters will contain zero-initialized elements) or to `reserve`? now that we are passing in a non-zero value, things are crashing.

Reply:
the intention is to `reserve`